tests/test_git_aliases.zunit

```diff
@@ -52,9 +52,24 @@
 
 @test 'git aliases exact match' {
    git config --global alias.st status
-   run _check_git_aliases "git status" "git status"
+￼  run _check_git_aliases "git status" "git status"
```

Comment:
seems like there is some weird white space here if you dont mind removing it

Reply:
Not sure what happened here, it wasn't a tab but showed up as same indent in my editor. Should be fixed now.